tests/utils/constants/constants.go

```diff
@@ -54,6 +54,9 @@ const (
 	MajorVersion              = "MAJOR_VERSION_ENV"
 	RHCSVersion               = "RHCS_VERSION"
 	RHCSSource                = "RHCS_SOURCE"
+	ModuleVersion             = "MODULE_VERSION"
+	ModuleSource              = "MODULE_SOURCE"
+	ModuleSourceLocal         = "MODULE_SOURCE_LOCAL"
```

Comment:
Why not having `MODULE_SOURCE` only and let the user decide whether local or not ?
is there any difference ?

Reply:
Basically the way terraform determines if the `source` option for a module is local or remote is purely based on whether or not the `version` option is set. So I set it up so that if the `MODULE_SOURCE_LOCAL` environment variable is set at all, then it will remove the `version` option so that you can use a local path in `MODULE_SOURCE`

Reply:
Ok. Can you add a comment for its usage ?

Reply:
Added comments for the three new environment variables